### PR TITLE
Bump versions to support ARM64

### PR DESCRIPTION
**Pull request by MatkovIvan**
* [ ] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests if this modifies the Windows code?
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Set minimal UWP version to 10.0.16299.0 and Microsoft.NETCore.UniversalWindowsPlatform to 6.2.x and .NET Framework to 4.6.1

## Related PRs or issues

[AB#74487](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/74487)

### CODE REVIEW

File: Tests/Microsoft.AppCenter.Crashes.Test.Windows/app.config [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-        <assemblyIdentity name="SQLitePCLRaw.core" publicKeyToken="1488e028ca7ab535" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-1.1.14.520" newVersion="1.1.14.520" />
+        <assemblyIdentity name="SQLitePCLRaw.core" publicKeyToken="1488e028ca7ab535" culture="neutral"/>
+        <bindingRedirect oldVersion="0.0.0.0-1.1.14.520" newVersion="1.1.14.520"/>
       </dependentAssembly>
     </assemblyBinding>
   </runtime>
-</configuration>
\ No newline at end of file
+<startup><supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1"/></startup></configuration>
```
Comment (guperrot) [inline Tests/Microsoft.AppCenter.Crashes.Test.Windows/app.config]: was that added manually or via UI? (formatting is odd)

File: Tests/Microsoft.AppCenter.Crashes.Test.Windows/packages.config [hunk 2]
```diff
@@ -20,18 +20,18 @@
   <package id="System.Diagnostics.Tracing" version="4.1.0" targetFramework="net452" />
   <package id="System.Globalization" version="4.0.11" targetFramework="net452" />
   <package id="System.IO" version="4.1.0" targetFramework="net452" />
-  <package id="System.IO.Compression" version="4.1.0" targetFramework="net452" />
+  <package id="System.IO.Compression" version="4.1.0" targetFramework="net452" requireReinstallation="true" />
```
Comment (guperrot) [inline Tests/Microsoft.AppCenter.Crashes.Test.Windows/packages.config]: what is that new flag? also I believe we need to update targetFramework here.

File: Apps/Contoso.Forms.Demo/Contoso.Forms.Demo.UWP/Contoso.Forms.Demo.UWP.csproj [hunk 3]
```diff
@@ -140,7 +140,7 @@
     <PackageReference Include="Microsoft.AppCenter.Distribute" Version="2.6.4" />
     <PackageReference Include="Microsoft.AppCenter.Push" Version="2.6.4" />
     <PackageReference Include="Xamarin.Forms" Version="4.4.0.991265" />
-    <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform" Version="6.2.9" />
+    <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform" Version="6.2.8" />
```
Comment (guperrot) [inline Apps/Contoso.Forms.Demo/Contoso.Forms.Demo.UWP/Contoso.Forms.Demo.UWP.csproj]: Any reason to downgrade? Is there an incompatibility we should document?
We can keep 1 app with latest version to check we are compatible with it if that was just for consistency. I think we updated demo on purpose and kept puppet on minimum.

File: SDK/AppCenter/Microsoft.AppCenter.WindowsDesktop/Microsoft.AppCenter.WindowsDesktop.csproj [hunk 4]
```diff
@@ -20,7 +20,7 @@
     <PackageReference Include="System.Management" Version="4.6.0" />
     <PackageReference Include="System.Configuration.ConfigurationManager" Version="4.6.0" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetFramework)'=='net45'">
+  <ItemGroup Condition="'$(TargetFramework)'=='net461'">
```
Comment (guperrot) [inline SDK/AppCenter/Microsoft.AppCenter.WindowsDesktop/Microsoft.AppCenter.WindowsDesktop.csproj]: Check if we use the `NET45` macro somewhere in code

File: Tests/Microsoft.AppCenter.Crashes.Test.Windows/app.config (hunk 1 above)
Comment (MatkovIvan) [reply Tests/Microsoft.AppCenter.Crashes.Test.Windows/app.config]: UI

File: Tests/Microsoft.AppCenter.Crashes.Test.Windows/packages.config (hunk 2 above)
Comment (MatkovIvan) [reply Tests/Microsoft.AppCenter.Crashes.Test.Windows/packages.config]: from UI..
I'll recheck it

File: Apps/Contoso.Forms.Demo/Contoso.Forms.Demo.UWP/Contoso.Forms.Demo.UWP.csproj (hunk 3 above)
Comment (MatkovIvan) [reply Apps/Contoso.Forms.Demo/Contoso.Forms.Demo.UWP/Contoso.Forms.Demo.UWP.csproj]: I'll update back

File: SDK/AppCenter/Microsoft.AppCenter.WindowsDesktop/Utils/DeviceInformationHelper.cs [hunk 5]
```diff
@@ -95,7 +95,7 @@ protected override string GetOsVersion()
 
         protected override string GetAppVersion()
         {
-#if NET45
+#if NET461
```
Comment (guperrot) [inline SDK/AppCenter/Microsoft.AppCenter.WindowsDesktop/Utils/DeviceInformationHelper.cs]: Does it work? We were not using the third digit before so just making sure.

Comment (MatkovIvan) [reply SDK/AppCenter/Microsoft.AppCenter.WindowsDesktop/Utils/DeviceInformationHelper.cs]: Yes

**Pull request merged**